README.md

```diff
@@ -277,6 +278,29 @@ validated_df = schema.validate(df)
 
 ```
 
+### Handling of Dataframe Columns not in the Schema
+
+By default, columns that aren't specified in the schema aren't checked. If you
+want to check that the dataframe *only* contains columns in the schema, specify
+`strict=True`:
+
+```python
+import pandas as pd
+from pandera import Column, DataFrameSchema, Int
+   
+schema = DataFrameSchema(
```

Comment:
I'll create a `CONTRIBUTING.md` doc in this repo with a suggested style-guide for formatting dataframes and dataframe schema... the formats I've come to are:

```
# If just specifying single column that can be
# expressed as a one-liner
DataFrameSchema({"col1": Column(...)})

# If just specifying one column in multiple lines, or multiple columns:
DataFrameSchema({
    "col1": Column(type, checks=[
        Check(...),
        Check(...),
    ]),
})

DataFrameSchema({
    "col1": Column(...),
    "col2": Column(...),
})

# if specifying columns with additional arguments
DataFrameSchema(
    {
        "col1": Column(...),
        "col2": Column(...),
    },
    strict=True)
```

Reply:
Good point.

I've updated the test to:
```
def test_dataframe_schema_strict():
    # checks if strict=True whether a schema error is raised because 'a' is not
    # present in the dataframe.
    schema = DataFrameSchema({"a": Column("int32", nullable=True)},
                             strict=True)
    df = pd.DataFrame({"b": [1, 2, np.nan]})
```

And I rebased and force pushed the changes so that there is only one commit so that the history remains clean.

In terms of styling it looks good. I wasn't sure about this:
```
# if specifying columns with additional arguments
DataFrameSchema(
    {
        "col1": Column(...),
        "col2": Column(...),
    },
    strict=True)
```
but after reading Pep8 I think it's the right solution, and after experimenting the best I came up with is this but it's less readable so think yours is right.:
```
# if specifying columns with additional arguments
DataFrameSchema({
    "col1": Column(...),
    "col2": Column(...),
}
    ,strict=True)
```

Reply:
To avoid any ambiguity, it might also make sense to include this too:
```
# if specifying single columns with additional arguments
DataFrameSchema({"a": Column(Int, nullable=True)},
                strict=True)
```

Reply:
Cool, I'll include this:
```
DataFrameSchema({"a": Column(Int, nullable=True)},
                strict=True)
```

I'll get to this in the coming week: https://github.com/cosmicBboy/pandera/issues/36

As always, your contributions are much appreciated! ✨🎉✨

Reply:
Thanks for `pandera`!

I think this PR has all the required changes so can be re-reviewed/merged.